src/Preview/Preview.tsx

```diff
@@ -704,60 +716,69 @@ const Preview: React.FC<PreviewProps> = ({
             }}
           >
-            <Tooltip
-              tooltipsize="small"
-              placement="bottom-start"
-              title={tooltipTexts.previousAsset}
-            >
-              <PreviousPreviewButton>
-                <StyledArrowButton
-                  data-testid={PreviewTestIds.PREVIEW_PREV_BUTTON}
-                  disabled={isPreviousDisabled}
-                  onClick={handlePreviousAsset}
-                >
-                  <ChevronLeft />
-                </StyledArrowButton>
-              </PreviousPreviewButton>
-            </Tooltip>
+            { !isVersionComparison && (
+              <Tooltip
+                tooltipsize="small"
+                placement="bottom-start"
+                title={tooltipTexts.previousAsset}
+              >
+                <PreviousPreviewButton>
+                  <StyledArrowButton
+                    data-testid={PreviewTestIds.PREVIEW_PREV_BUTTON}
+                    disabled={isPreviousDisabled}
+                    onClick={handlePreviousAsset}
+                  >
+                    <ChevronLeft />
+                  </StyledArrowButton>
+                </PreviousPreviewButton>
+              </Tooltip>
+            )}
             <>
               {(isCurrentAssetReady === false && reactComponent === undefined) && (
               <CircularProgressContainer
                 data-testid={PreviewTestIds.PREVIEW_CIRCULAR_PROGRESS}
                 container
                 sx={{
-                  height: `calc(100% - ${isVideo ? '55' : '61'}px)`,
+                  height: `calc(100% - ${isVideo ? '55' : `${isVersionComparison ? '0' : '66'}`}px)`,
                 }}
               >
                 <CircularProgress />
               </CircularProgressContainer>
               )}
               {renderOptions()}
             </>
-            <Tooltip
-              tooltipsize="small"
-              placement="bottom-end"
-              title={tooltipTexts.nextAsset}
-            >
-              <NextPreviewButton>
-                <StyledArrowButton
-                  data-testid={PreviewTestIds.PREVIEW_NEXT_BUTTON}
-                  disabled={isNextDisabled}
-                  onClick={handleNextAsset}
-                >
-                  <ChevronRight />
-                </StyledArrowButton>
-              </NextPreviewButton>
-            </Tooltip>
+            { !isVersionComparison && (
+              <Tooltip
+                tooltipsize="small"
+                placement="bottom-end"
+                title={tooltipTexts.nextAsset}
+              >
+                <NextPreviewButton>
+                  <StyledArrowButton
+                    data-testid={PreviewTestIds.PREVIEW_NEXT_BUTTON}
+                    disabled={isNextDisabled}
+                    onClick={handleNextAsset}
+                  >
+                    <ChevronRight />
+                  </StyledArrowButton>
+                </NextPreviewButton>
```

Comment:
maybe a wrapper span is needed around the button to prevent this error from showing the enduser's console @jcmcardama 

<img width="1792" alt="Screenshot 2024-09-25 at 7 27 43 PM" src="https://github.com/user-attachments/assets/9a98d400-190d-4ebc-b9fd-e26c5d749b9d">